packages/bot-web-ui/src/components/dashboard/dashboard.scss

```diff
@@ -495,9 +510,8 @@
     &__media {
         background: var(--general-section-1);
         text-align: center;
-        padding: 0.8rem;
         width: 100%;
-        height: 50%;
+        height: 55%;
         margin-bottom: 0.8rem;
```

Comment:
@maryia-matskevich-deriv since we're removing the padding: 0.8rem, can we increase the margin-bottom from 0.8rem to 1.6rem please as the text looks too close to the media now. Thanks.

Reply:
@farabi-deriv could you please show it? As you can see on my recording video which is attached to this PR it looks ok for iPhone 12mini in my opinion hm

Reply:
@maryia-matskevich-deriv You can check app.deriv.com/bot and the test link on your phone. The spacing between the media and the text has decreased since removing the padding 0.8. Hence suggested to update margin-bottom to 1.6rem. Then it will remain same as current production

Reply:
which device did you check?